hack/.deployUtils

```diff
@@ -322,86 +311,82 @@ deployPrometheusForFederation() {
   fi
 }
 
-cleanup
-
-port80=9090
-port443=8445
-proxyPort=9200
-metalLBSubnetStart=200
-
-# Create network for the clusters
-docker network create -d bridge --subnet 172.32.0.0/16 mgc --gateway 172.32.0.1 \
-  -o "com.docker.network.bridge.enable_ip_masquerade"="true" \
-  -o "com.docker.network.driver.mtu"="1500"
-
-# Create Kind control plane cluster
-kindCreateCluster ${KIND_CLUSTER_CONTROL_PLANE} ${port80} ${port443}
-
-# Deploy the submariner broker to cluster 1
-deploySubmarinerBroker ${KIND_CLUSTER_CONTROL_PLANE}
-
-# Join cluster 1 to the submariner broker
-joinSubmarinerBroker ${KIND_CLUSTER_CONTROL_PLANE}
-
-# Install the Gateway API CRDs in the control cluster
-installGatewayAPI ${KIND_CLUSTER_CONTROL_PLANE}
-
-# Deploy ingress controller
-deployIngressController ${KIND_CLUSTER_CONTROL_PLANE}
 
-# Deploy cert manager
-deployCertManager ${KIND_CLUSTER_CONTROL_PLANE}
 
-# Deploy argo cd
-deployArgoCD ${KIND_CLUSTER_CONTROL_PLANE}
+# quickstart-setup specific functions
 
-# Deploy Dashboard
-deployDashboard $KIND_CLUSTER_CONTROL_PLANE 0
-
-# Add the control plane cluster
-argocdAddCluster ${KIND_CLUSTER_CONTROL_PLANE} ${KIND_CLUSTER_CONTROL_PLANE}
-
-# Initialize local dev setup for the controller on the control-plane cluster
-initController ${KIND_CLUSTER_CONTROL_PLANE}
-
-# Deploy OCM hub
-deployOCMHub ${KIND_CLUSTER_CONTROL_PLANE}
-
-# Deploy Redis
-deployRedis ${KIND_CLUSTER_CONTROL_PLANE}
+configureController() {
+    clusterName=${1}
+    kubectl config use-context kind-${clusterName}
+    echo "Initialize local dev setup for the controller on ${clusterName}"
 
-# Deploy MetalLb
-deployMetalLB ${KIND_CLUSTER_CONTROL_PLANE} ${metalLBSubnetStart}
+    cat <<EOF | kubectl apply -f -
```

Comment:
This one is still for the quickstart script only. I've moved it all anyway: this script will be sourced anyway because of the shared functions and moving everything here makes it "one place to hold them all". Don't have a strong opinion on where to hold quickstart and local-setup specific things. And not sure I could avoid having this bit at all